akka-docs-new/src/main/paradox/guide/modules.md

```diff
@@ -0,0 +1,165 @@
+same. The patterns that you use locally on multi-core systems translate directly to remote systems. 
+You will rarely need to use Remoting directly, but it provides the foundation on which the Cluster subsystem is built.
+ 
+Some of the problems Remoting solves are
+
+* How to address actor systems living on remote hosts?
+* How to address individual actors on remote actor systems?
+* How to turn messages to bytes on the wire?
+* How to manage low-level, network connections (and reconnections) between hosts, detect crashed actor systems and hosts, 
+  all transparently?
+* How to multiplex communications from unrelated set of actors on the same network connection, all transparently?
+
+### Cluster
+
+If you have a set of actor systems that cooperate to solve some business problem, then you likely want to manage these set of 
+systems in a disciplined way. While Remoting solves the problem of addressing and communicating with components of 
+remote systems, Clustering gives you the ability to organize these into a "meta-system" tied together by a membership
+protocol. **In most of the cases, you want to use the Cluster module instead of using Remoting directly.** 
+Cluster provides an additional set of services on top of Remoting that most of the real world applications need. 
+
+The problems the Cluster module solves (among others) are
+
+* How to maintain a set of actor systems (a cluster) that can communicate with each other and consider each other as part of the cluster?
+* How to introduce a new system safely to the set of already existing members?
+* How to detect reliably systems that are temporarily unreachable?
+* How to remove failed hosts/systems (or scale down the system) so that all remaining members agree on the remaining subset of the cluster?
+* How to distribute computations among the current set of members?
+* How do I designate members of the cluster to a certain role, in other words to provide certain services and not others?
+
+### Persistence
+
+Just like objects in OOP actors keep their state in volatile memory. Once the system is shut down, gracefully or
+because of a crash, all data that was in memory is lost. Persistence provide patterns to enable actors to persist 
+events that lead to their current state. Upon startup events can be replayed to restore the state of the entity hosted 
+by the actor. The event stream can be queried and fed into additional processing pipelines (an external Big Data 
+cluster for example) or alternate views (like reports).
+ 
+Persistence tackles the following problems:
+ 
+* How do I restore the state of an entity/actor when system restarts or crashes?
+* How do I implement a [CQRS system](https://msdn.microsoft.com/en-us/library/jj591573.aspx)?
+* How do I ensure reliable delivery of messages in face of network errors and system crashes?
+* How do I introspect domain events that has lead an entity to its current state?
+
+### Cluster Singleton
```

Comment:
change the order of the cluster modules, with most important first:
* Cluster Sharding
* Distributed Data
* Cluster Singleton
* Cluster Publish-Subscribe